### ISSUE TITLE
Modal Container Width is hardcoded

### ISSUE THREAD

**Issue by miljan-aleksic** [uikit/uikit#3221]
The [docs](https://getuikit.com/docs/modal#container-modifier) states that `uk-modal-container` class will expand the modal dialog to the default Container width. But the class value is [hardcoded](https://github.com/uikit/uikit/blob/develop/src/less/components/modal.less#L41).

I am assuming the docs are right and the code should read the width from the container component. What I am not sure if that should be done in the base less or in the theme less files.

**Comment by zzseba78** [comment]
I was surprised when I see the pixel value too instead of container width variable... uikit is very well structured, maybe the idea it to avoid dependency on the Container component ( i know it doesn´t make much sense ), and `uk-modal-container` just act as a width modifier.

**Comment by miljan-aleksic** [comment]
Yes, that could be the reason but then there should exist a global variable used by both components.
